src/main/java/blackjack/domain/gambler/Name.java

```diff
@@ -0,0 +1,47 @@
+package blackjack.domain.gambler;
+
+import static blackjack.view.ErrorMessage.INVALID_PLAYER_NAME_FORMAT;
+import static blackjack.view.ErrorMessage.INVALID_PLAYER_NAME_LENGTH;
+
+import java.util.Objects;
+
+public class Name {
+    private static final int MAX_NAME_LENGTH = 5;
+    private static final int MIN_NAME_LENGTH = 2;
+    private static final String SPACE = " ";
+
+    private final String name;
+
+    public Name(final String name) {
+        this.name = validateForm(name);
+    }
+
+    private String validateForm(final String name) {
+        if (name.length() < MIN_NAME_LENGTH || name.length() > MAX_NAME_LENGTH) {
+            throw new IllegalArgumentException(INVALID_PLAYER_NAME_LENGTH.getMessage());
+        }
+        if (name.split(SPACE).length != 1) {
```

Comment:
저는 아래 방식이 좀 더 가독성이 좋다고 느껴지네요 ㅎㅎ

```suggestion
        if (name.contains(SPACE)) {
```

Reply:
앗 저도 그렇게 생각합니다..!
수정했습니다.
f2d82cffeba3813cad870e70893f24afc6eed97d